amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/README.md

```diff
@@ -5,22 +5,60 @@ This workflow automates and standardizes the re-processing of RNASeq datasets fr
 ### Dependencies 
 * Docker
 * A CWL execution engine ([cwltool](https://github.com/common-workflow-language/cwltool) or [toil](https://toil.readthedocs.io/en/latest/))
+
 ### Usage
-Users must modify the *main.json* file to point to their synapse config file.  In addition, they must provide an array of synapse ID's that correspond to the BAM files that they would like to process.  
+Whichever tool you choose to use, you will need to create or symlink a 
+[synapse config file](https://docs.synapse.org/articles/client_configuration.html#customize-the-synapse-configuration-file)
+in this directory.
 
-#### cwltool execution 
-`cwl-runner main-paired.cwl main.json`
+Subfolders under the `jobs` folder supply the following
+* `options.json`, used in `run-toil.py`, see instructions below
+* `job.json`, used to supply the cwl arguments, regardless of tool choice
+* requirements files that supply the resource requirements to CWL workflows
 
-#### toil execution 
+#### cwltool execution
+The following instructions are for running the `cwl-runner` in BASH. In the
+example below, the job directory "jobs/test-main-paired" is used.
 
-- ssh to toil cluster leader node
-- modify `run-toil.sh` to specify resource requests
-- execute toil run script:
 ```bash
-chmod +x run-toil.sh
-./run-toil.sh
+# Create symlinks to resource files
+links=$(utils/linkresources.py jobs/test-main-paired)
+
+# Use cwltool to run a workflow
+cwl-runner main-paired.cwl jobs/test-main-paired/job.json
+
+# Remove symlinks
+utils/unlinkresources.py $links
```

Comment:
This is not ideal but couldn't find a better way to deal with the complexity. Thoughts?

Reply:
why you need symlink in the first place?

Reply:
Limitations of CWL and wanting to use workflows with a variety of different resource files